.github/ISSUE_TEMPLATE/nova_tarefa.md

```
@@ -0,0 +1,40 @@
+### Título da Tarefa
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)